src/main/cpp/states/shooter/ShooterStateAutoHigh.cpp

```diff
@@ -66,7 +66,7 @@ void ShooterStateAutoHigh::Init()
        }
        if (currentLeftIndexerStateMgrState == IndexerStates::INDEXER_STATE::INDEX)
        {
-            m_shooterTarget = 0.1939*inches + 40.788 - 0.25;
+            m_shooterTarget = 0.1939*inches + 40.788;

```

Comment:
Did you mean to change ShooterStateAutoHigh?

Reply:
Yes, this was a change from Pittsburg that I noticed wasn't committed. However you can disregard this change since it is all changing anyway.